jax/numpy/__init__.py

```diff
@@ -431,7 +431,7 @@ def _init():
   from jax._src import util
   # Builds a set of all unimplemented NumPy functions.
   for name, func in util.get_module_functions(np).items():
-    if name not in globals():
+    if name not in globals() and not name.startswith('_'):
```

Comment:
Should this be name.startswith('_') without the `not`?

Reply:
No, I think it's correct as written. We don't want to add `name` to globals if it starts with an underscore.